src/main/kotlin/no/nav/amt/person/service/nav_bruker/NavBrukerUpdateJob.kt

```diff
@@ -0,0 +1,27 @@
+package no.nav.amt.person.service.nav_bruker
+
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock
+import no.nav.common.job.JobRunner
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+import java.time.LocalDateTime
+
+@Component
+class NavBrukerUpdateJob(
+	private val navBrukerService: NavBrukerService
+) {
+
+	@Scheduled(cron = "@midnight")
+	@SchedulerLock(name = "navBrukerUpdater", lockAtMostFor = "60m")
+	fun update() {
+		JobRunner.run("oppdater_nav_brukere") { oppdaterBrukere()}
+	}
+
+	private fun oppdaterBrukere() {
+		val brukere = navBrukerService.get(offset = 0, limit = 500, syncedSince = LocalDateTime.now().minusDays(3))
```

Comment:
Leser jeg det riktig at vi bare skal oppdatere 500 brukere om dagen? Da vil det i allefall ta noen år før vi har oppdatert alle 😅

Betyr `syncedSince` brukere som sist var synkronisert for mins n dager siden? Jeg tror ikke jeg ville helt ha forstått hva det var til uten å lese sqlen. Vanskelig å komme på noe godt navn, beste alternativ jeg har er `lastSyncOlderThan` eller noe.

Reply:
Hahaha, ja, det kan stemme ja, vil ta 1200 dager! Kanskje jeg skal lage en internal controller som er mer agressiv så man kommer igjennom alle en gang litt fortere også kan den schedulerte jobben være som den er eller kan kanskje prøve å doble antallet elns

Reply:
Ordlyden ble kanskje feil i parameteret når jeg tenker meg om. Det er jo egentlig "notSyncedSince" siden den henter de som ikke har vært syncronisert på 3 dager